### ISSUE TITLE
[themes] Solarized Dark selection color has no enough contrast

### ISSUE THREAD

**Issue by siburuxue** [Microsoft/vscode#48106]
Issue Type: <b>Feature Request</b>

在使用Solarized Dark主题时，当选中一些文字之后，被选中的文字的背景颜色和主题背景色的颜色反差太小，导致根本看不清这些文字是否被选中

VS Code version: Code 1.22.2 (3aeede733d9a3098f7b4bdc1f66b63b0f48c1ef9, 2018-04-12T16:32:53.389Z)
OS version: Darwin x64 17.5.0

**Comment by MaleDong** [comment]
@siburuxue : Can you show us a mp4 in a zip file by submitting it to re-produce the issue?

**Comment by ramya-rao-a** [comment]
@siburuxue Please provide a screenshot of the problem

**Comment by siburuxue** [comment]
[test.mp4.zip](https://github.com/Microsoft/vscode/files/1927604/test.mp4.zip)
这个是我录制的视频

**Comment by MaleDong** [comment]
@siburuxue ：Thanks. But I cannot open the mp4 to see what's happening……
And what's more, it seems that Microsoft member has opened that to analyze the problem……Let's wait for them:)

**Comment by siburuxue** [comment]
@MaleDong 可能是我之前录的内容解释的不详细 我重新录了一遍 地址：http://121.42.230.254/mov.zip
